pages/common/ssh-keygen.md

```diff
@@ -22,6 +22,10 @@
 
 `ssh-keygen -l -F {{remote_host}}`
 
+- Remove the keys of a host from your known_hosts file (useful when a known host has a new key):
+
+`ssh-keygen -R {{myhost.com}}`
```

Comment:
s/myhost.com/remote_host

Reply:
ah, I thought I had already changed that :)